app/assets/javascripts/views/shared/footer.js

```diff
@@ -0,0 +1,35 @@
+(function (App) {
+  App.View.Footer = Backbone.View.extend({
+
+    el: '.c-footer',
+
+    initialize: function () {
+      this._setVars();
+      this._setEventListeners();
+    },
+
+    _setVars: function () {
+      this.body = document.querySelector('body');
+      this.refElem = this.el.querySelector('.js-toggle-tooltip');
+      this.socialTooltip = new App.View.SocialTooltip({ refElem: this.refElem });
+    },
+
+    _setEventListeners: function () {
+      this.refElem.addEventListener('mouseenter', function () {
+        if (this.body.contains(this.socialTooltip.el)) {
```

Comment:
Yeah, all this kind of methods need a deep review... I did a "make it work" version originally.